### DISCUSSION TITLE
Portuguese Translation

### DISCUSSION THREAD

**Discussion by spac3h** [code4recovery/12-step-meeting-list#1451] [Q&A]
Hi all, i am thinking of translating the plugin to Portuguese, I already translated the old UI for our needs using and hard coding approach. I Would like to translate both the TSML UI and the Legacy UI officially. Can you point me on the right direction?

**Comment by joshreisner** [comment]
great!

if you could translate the quoted strings that are not surrounded by % in this file: https://github.com/code4recovery/tsml-ui/blob/main/src/i18n/en.ts

~also add `pt` strings in this file: https://github.com/code4recovery/spec/blob/main/src/types.php~

EDIT for the future - the files in this folder: https://github.com/code4recovery/spec/tree/main/data

if you want to just upload them to this discussion i can handle the rest, thanks!

**Comment by spac3h** [comment]
Hi, this is now translated. Once we can use the PT language please give me some details on how to do it.
[files.zip](https://github.com/user-attachments/files/15945705/files.zip)
Thanks

**Comment by joshreisner** [comment]
great thanks! TSML UI should now be translated ✅ 

i'll ask the team on the best approach to translating legacy

**Reply by spac3h** [reply]
How do i use the translated version of TSML UI while i translate the Legacy version? Do i need to reinstall or update the plugin?

**Reply by joshreisner** [reply]
if your browser language is set to portuguese you should see it

<img width="1308" alt="Screenshot 2024-06-24 at 7 12 45 AM" src="https://github.com/code4recovery/12-step-meeting-list/assets/1551689/a52217a0-7ca0-4d7c-b060-99ef92852a6d">
